pkg/controller/migration/convert/network.go

```diff
@@ -496,14 +495,14 @@ func handleIPPools(c *components, install *Installation) error {
 	}
 
 	// Convert any found CRD pools into Operator pools and add them.
-	if v4pool != nil {
+	if render.GetIPv4Pool(install.Spec.CalicoNetwork.IPPools) == nil && v4pool != nil {
```

Comment:
This and the change below are to ensure we don't add a pool if one already exists for that version